src/js/components/Meter/Circle.js

```diff
@@ -101,15 +101,20 @@ const Circle = forwardRef((props, ref) => {
         }
         pathCaps.unshift(pathCap);
       } else {
-        const y = type === 'semicircle' ? width : width / 2;
-        const d = arcCommands(width / 2, y, radius, startAngle, endAngle);
+        const d = arcCommands(
+          width / 2,
+          width / 2,
```

Comment:
This should be "width" for the semicircle case, part of why I think "centerY" would make it clearer.

Reply:
Also, to catch this, I think we should have unit tests for both "round" and "not round" cases for each type.

Reply:
So I have `centerY` which is width/2 for the `semicircle` this is working on my end if I just put the width then I get the below picture.. 
Im not sure if Im missing something that you are seeing on your end but with the lastest changes the `semicircle` looks correct. 

<img width="533" alt="Screen Shot 2021-08-03 at 8 02 01 PM" src="https://user-images.githubusercontent.com/42451602/128109972-5121c368-d190-40a7-b525-a57159b25459.png">